clio-server/src/main/scala/org/broadinstitute/clio/server/service/WgsCramService.scala

```diff
@@ -31,7 +35,8 @@ class WgsCramService(
   ): Future[UpsertId] = {
     val updatedTransferMetadata = transferMetadata.copy(
       documentStatus =
-        transferMetadata.documentStatus.orElse(Some(DocumentStatus.Normal))
+        transferMetadata.documentStatus.orElse(Some(DocumentStatus.Normal)),
+      regulatoryDesignation = Some(RegulatoryDesignation.ResearchOnly)
```

Comment:
Same here.

Reply:
You're right, that will mean we don't have to make a change in the future.